### PR TITLE
Add setters to TLE class

### PR DESCRIPTION
**Pull request by Derollez**
- Add setter functions to be able to modify TLE in place. Useful for LEOPs when TLE object is still subject to changes and needs to be updated.

### CODE REVIEW

**Review by lucas-bremond — APPROVED** [review]
LGTM!!

File: test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/SGP4/TLE.test.cpp [hunk 1]
```diff
@@ -651,6 +651,260 @@ TEST (OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_SGP4_TLE, getSecond
...
+        const String newFirstLine = "1 25544U 98067A   87051.85104286 -.00002182  00000-0 -11606-4 0  2924" ;
+
+        EXPECT_EQ(newFirstLine, tle.getFirstLine()) ;
+        EXPECT_EQ(secondLine, tle.getSecondLine()) ;
+
+    }
+
+}
+
+TEST (OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_SGP4_TLE, SetRevolutionNumerAtEpoch)
```
Comment (robinpdm) [inline test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/SGP4/TLE.test.cpp]: Typo

File: src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/SGP4/TLE.cpp [hunk 2]
```diff
@@ -372,6 +372,115 @@ Integer                         TLE::getSecondLineChecksum                  ( )
...
+        throw ostk::core::error::runtime::Undefined("TLE") ;
+    }
+
+    if (aRevolutionNumberAtEpoch > 99999 || aRevolutionNumberAtEpoch.isNegative())
+    {
+        throw ostk::core::error::runtime::Wrong("Provided Revolution Number at Epoch") ; 
+    }
+
+    const String revolutionNumberAtEpochString = aRevolutionNumberAtEpoch.toString() ;
+    const String sanitizedrevolutionNumberAtEpochString = String::Replicate("0", 5 - revolutionNumberAtEpochString.getLength()) + revolutionNumberAtEpochString ;
```
Comment (robinpdm) [inline src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/SGP4/TLE.cpp:470]: `sanitizedRevolutionNumberAtEpochString`

**Review by robinpdm — APPROVED** [review]
LGTM too!

**Pull request merged**